pinot-plugins/pinot-file-system/pinot-adls/pom.xml

```diff
@@ -39,12 +39,46 @@
     <dependency>
       <groupId>com.microsoft.azure</groupId>
       <artifactId>azure-data-lake-store-sdk</artifactId>
-      <version>2.1.5</version>
+      <version>2.3.9</version>
     </dependency>
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-storage-file-datalake</artifactId>
-      <version>12.0.0-beta.12</version>
+      <version>12.4.0</version>
+    </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-identity</artifactId>
+      <version>1.2.2</version>
     </dependency>
   </dependencies>
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.codehaus.woodstox</groupId>
```

Comment:
What's the minimal do you think it's needed for your change? Are the following dependencies going to be used in the real code, or they are used in the test? If latter, maybe we can restrict the scope of those dependencies.

Reply:
I need all of these as. azure-identity pulls in and will lead to convergence error